desktop.blocks/page/__conditional-comment/page__conditional-comment.bemhtml

```diff
@@ -0,0 +1,23 @@
+block('page').elem('conditional-comment')(
+    tag()(false),
+
+    content()(function() {
+        var ctx = this.ctx,
+            cond = ctx.condition
+                .replace('<', 'lt')
+                .replace('>', 'gt')
+                .replace('=', 'e'),
+            hasNegation = cond.indexOf('!') > -1,
+            includeOthers = ctx.includeOthers,
```

Comment:
@narqo 
should be switched on by default?
I mean is `msieOnly` equal `true` if not set `false` explicitly? or we should treat `undefined` as `false` as well?